src/components/Item/Item.js

```diff
@@ -9,20 +11,65 @@ import TotalPrice from './TotalPrice';
 import DiscountPrice from './DiscountPrice';
 import PriceBox from './PriceBox';
 import Wrapper from './Wrapper';
+import Details from './Details';
 
-function Item() {
+const getTotalPrice = function(item) {
+  const { quantity, price } = item;
+  return quantity * (price.regular - price.discount);
+};
+
+const renderAttributes = function(item) {
+  const { variants } = item;
+
+  if (!variants) {
+    return null;
+  }
+
+  const size = variants.size ? variants.size : NO_SIZE;
+  const color = variants.color ? variants.color : NO_COLOR;
+
+  return (
+    <>
+      <Attribute>Size: {size}</Attribute>
+      <Attribute>Color: {color}</Attribute>
+    </>
+  );
+};
+
+const renderPrices = function(item) {
+  if (!item.price) {
+    return null;
+  }
+  const { price } = item;
+
+  return (
+    <PriceBox>
+      <DiscountPrice>Sale ${price.discount}</DiscountPrice>
+      <Price>Regular ${price.regular}</Price>
+      <TotalPrice>Total ${getTotalPrice(item)}</TotalPrice>
```

Comment:
I would pass only needed values to that function instead of whole `item` object.